### ISSUE TITLE
Old Rhythmbox support is broken

### ISSUE THREAD

**Issue by GoogleCodeExporter** [memeco/osd-lyrics#236]
```
What steps will reproduce the problem?
请列出导致问题的操作
1.Open osd-lyrics
2.osd-lyrics launches rhythmbox
3.osd-lyrics shows the windows asking for a player to launch (suported players: 
mpd & rhythmbox; see attached screenshot) and it keeps doing the same operation 
over and over not showing the lyrics ever after playing the song with rhythmbox

What's the output in terminal (run with command `osd-lyrics -d debug')?
请给出在终端下运行的输出（使用“osd-lyrics -d
debug”来运行）

See attached file osdlyrics_debug.txt for full report

*Debug*: in function ol_player_get_active_player: ol_player.c[137]
trying Gmusicbrowser
*Debug*: in function ol_player_gmusicbrowser_init_dbus: 
ol_player_gmusicbrowser.c[279]
get proxy failed: Could not get owner of name 'org.gmusicbrowser': no such name
*Debug*: in function ol_player_get_active_player: ol_player.c[137]
trying MPRIS
*Debug*: in function ol_player_get_active_player: ol_player.c[137]
trying MPRIS2
*Debug*: in function ol_player_get_active_player: ol_player.c[137]
trying MOC
^C


What's the player and which version you are using?
您使用的播放器是什么？它的版本号是多少？

Rhythmbox 0.12.8

What's version of OSD Lyrics?
OSD Lyrics的版本是多少？

What's your Linux distro?
您使用哪种发行版？

Ubuntu 10.04.3 LTS

Linux 2.6.32-33-generic #72-Ubuntu SMP Fri Jul 29 21:07:13 UTC 2011 x86_64 
GNU/Linux

```

Original issue reported on code.google.com by `rop...@gmail.com` on 8 Sep 2011 at 1:55

Attachments:
- [Choose_a_player_to_launch.png](https://storage.googleapis.com/google-code-attachments/osd-lyrics/issue-236/comment-0/Choose_a_player_to_launch.png)
- [osdlyrics_debug.txt](https://storage.googleapis.com/google-code-attachments/osd-lyrics/issue-236/comment-0/osdlyrics_debug.txt)

**Comment by GoogleCodeExporter** [comment]
```
osd-lyrics version 0.4.1-1~lucid1
```

Original comment by `rop...@gmail.com` on 8 Sep 2011 at 2:03

**Comment by GoogleCodeExporter** [comment]
```
Did you enable the MPRIS plugin in Rhythmbox?
```

Original comment by `tigerso...@gmail.com` on 9 Sep 2011 at 4:04

**Comment by GoogleCodeExporter** [comment]
```
Any solution?
```

Original comment by `JuanDav...@gmail.com` on 18 Sep 2011 at 4:18

**Comment by GoogleCodeExporter** [comment]
```
Since 0.4.1, OSD Lyrics use MPRIS2 protocol to control Rhythmbox instead of the 
old control interface.
However, it seems old versions of Rhythmbox does not support MPRIS2.
The old rhythmbox support will be back in 0.4.2. You can install git version if 
you can't wait for it.
```

Original comment by `tigerso...@gmail.com` on 19 Oct 2011 at 7:15
- Changed title: **Old Rhythmbox support is broken**
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Issue 248 has been merged into this issue.
```

Original comment by `tigerso...@gmail.com` on 25 Nov 2011 at 2:32

**Comment by GoogleCodeExporter** [comment]
```
OSD Lyrics 0.4.2 beta1 is out, which should fix the support of Rhythmbox prior 
to 0.13. Please try is version.
```

Original comment by `tigerso...@gmail.com` on 25 Nov 2011 at 2:33

**Comment by GoogleCodeExporter** [comment]
```
Issue 252 has been merged into this issue.
```

Original comment by `tigerso...@gmail.com` on 14 Dec 2011 at 3:53

**Comment by GoogleCodeExporter** [comment]
```
its solved by upgrading the Rhythmbox to version 0.13.1
```

Original comment by `z...@elektro08.com` on 6 Nov 2012 at 8:23
